src/mono/wasm/runtime/loader/assets.ts

```diff
@@ -415,3 +415,19 @@ export function cleanupAsset(asset: AssetEntryInternal) {
     asset.pendingDownload = null as any; // GC
     asset.buffer = null as any; // GC
 }
+
+export function appendUniqueQuery(attemptUrl: string): string {
+    if (loaderHelpers.assetUniqueQuery) {
+        attemptUrl = attemptUrl + loaderHelpers.assetUniqueQuery;
+    }
+
+    return attemptUrl;
+}
+
+export function toAbsoluteBaseUri(path: string) {
+    if ("URL" in globalThis) {
+        return new URL(path, document.baseURI).toString();
```

Comment:
You probably also need to check that `document` exists. I'm not sure what should happen on WebWorker